### DISCUSSION TITLE
Provide arm Docker Container (arm64/v8)

### DISCUSSION THREAD

**Discussion by dtcooper** [tiangolo/nginx-rtmp-docker#88] [Questions] [answered]
Hi there, now that Docker is available for Apple's M1 chipset, a native container that runs on `linux/arm64/v8` would be fantastic! Thanks for considering it.

Cheers,

David

**Comment by jNaimXIII** [comment]
Has this idea been abandoned? Would be nice to have an arm build.

**Comment by ramiel** [comment]
Any news on this? There's also a PR about this #32

**Comment by tiangolo** [comment, marked as answer]
This was covered in https://github.com/tiangolo/nginx-rtmp-docker/pull/65 a while ago, there are now arm64 images too. :rocket: :tada:
